regression/smt2_solver/quantifiers/bv-quantifier-valid.desc

```diff
@@ -0,0 +1,7 @@
+KNOWNBUG
+bv-quantifier-valid.smt2
+
+^EXIT=0$
+^SIGNAL=0$
+^unsat$
+--
```

Comment:
It would be helpful for either this file or the commit message to include an explanation what exactly is missing here.